cal.js

```diff
@@ -0,0 +1,67 @@
+const num = document.querySelector(".num1");
+const num1 = document.querySelector(".num2");
+const num2 = document.querySelector(".num3");
+const num3 = document.querySelector(".num4");
+const num4 = document.querySelector(".num5");
+const num5 = document.querySelector(".num6");
+const num6 = document.querySelector(".num7");
+const num7 = document.querySelector(".num8");
+const num8 = document.querySelector(".num9");
+const num9 = document.querySelector(".num10");
+const num10 = document.querySelector(".num11");
+const num11 = document.querySelector(".num12");
+// const num12 = document.querySelector('.num13');
+const num13 = document.querySelector(".num14");
+const num14 = document.querySelector(".num15");
+const num15 = document.querySelector(".num16");
+const display = document.querySelector(".display");
+
+num.addEventListener("click", () => {
+  display.append(num.innerHTML);
+});
+
+num1.addEventListener("click", () => {
+  display.append(num1.innerHTML);
+});
+num2.addEventListener("click", () => {
+  display.append(num2.innerHTML);
+});
+num3.addEventListener("click", () => {
+  display.append(num3.innerHTML);
+});
+num4.addEventListener("click", () => {
+  display.append(num4.innerHTML);
+});
+num5.addEventListener("click", () => {
+  display.append(num5.innerHTML);
+});
+num6.addEventListener("click", () => {
+  display.append(num6.innerHTML);
+});
+num7.addEventListener("click", () => {
+  display.append(num7.innerHTML);
+});
+num8.addEventListener("click", () => {
+  display.append(num8.innerHTML);
+});
+num9.addEventListener("click", () => {
+  display.append(num9.innerHTML);
+});
+num10.addEventListener("click", () => {
+  display.append(num10.innerHTML);
+});
+num11.addEventListener("click", () => {
+  display.append(num11.innerHTML);
+});
+// num12.addEventListener('click', () => {
```

Comment:
Please remove this comment if its not being used.

Reply:
please which one in currently used.